src/frontend/Komponenter/Behandling/VedtakOgBeregning/Overgangsstønad/Opphør/Opphør.tsx

```diff
@@ -14,12 +14,22 @@ import { EnsligTextArea } from '../../../../../Felles/Input/TekstInput/EnsligTex
 import { VEDTAK_OG_BEREGNING } from '../../Felles/konstanter';
 import { AlertError } from '../../../../../Felles/Visningskomponenter/Alerts';
 import { Button } from '@navikt/ds-react';
+import { AGray50 } from '@navikt/ds-tokens/dist/tokens';
 import { useRedirectEtterLagring } from '../../../../../App/hooks/felles/useRedirectEtterLagring';
 import { v4 as uuidv4 } from 'uuid';
 
-const StyledFormElement = styled.div`
-    margin-top: 2rem;
-    margin-bottom: 2rem;
+const Container = styled.div`
+    margin-top: 1rem;
+    padding: 1rem;
+    background-color: ${AGray50};
+`;
```

Comment:
Samme som før hehe: kan bruke gap for å lage spacing

Reply:
Noterer ned til senere PR 👍